cmd/digger/main.go

```diff
@@ -238,7 +239,8 @@ func gitHubCI(lock core_locking.Lock, policyChecker core_policy.Checker, backend
 		if err != nil {
 			reportErrorAndExit(githubActor, fmt.Sprintf("Failed to process GitHub event. %s", err), 6)
 		}
-		logImpactedProjects(impactedProjects, prNumber)
```

Comment:
Lets use a more descriptive function name. We can call it `getImpacagedProjectsAsString`

Reply:
Done